code/go/0chain.net/smartcontract/stakepool/spenum/enums.go

```diff
@@ -9,7 +9,8 @@ func init() {
 type Provider int
 
 const (
-	Miner Provider = iota + 1
+	Invalid Provider = iota
```

Comment:
we don't need it, simply when get from map check whether status is found or not

Reply:
I was added to give [ToProviderType](https://github.com/0chain/0chain/blob/000ff47a73fe23f85be4318ed2aee47bc0e6c5a2/code/go/0chain.net/smartcontract/stakepool/spenum/enums.go#L39) something to return if called with an invalid string.